### PR TITLE
Only open deep links in Android [mozilla/webmaker-core#554]

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
@gvn R?

File: src/components/app-cta/app-cta.jsx [hunk 1]
```diff
@@ -36,9 +43,15 @@ var AppCta = React.createClass({
     this.setState({
       loading: true
     });
+
+    if (!this.isAndroid()) {
+      return window.location = AppCta.FALLBACK_URL;
+    }
+
     this.timeout = setTimeout(function () {
       window.location = AppCta.FALLBACK_URL;
```
Comment (gvn) [inline src/components/app-cta/app-cta.jsx]: Looks like the play store URL still opens in the background once you re-focus back to your browser on an Android device. Seems like that shouldn't be the case...

**Comment by gvn** [comment]
Should probably say "Get App" for non-Android users...or something.

**Comment by gvn** [comment]
Looks like the play store URL still opens in the background once you re-focus back to your browser on an Android device. Seems like that shouldn't be the case...I'm not sure how to get around that though. :imp:

**Comment by gvn** [comment]
Also, needs a quick rebase...

**Comment by gvn** [comment]
Also also: I don't seem to be able to load user1/project1...eg: Hit "Open App" from `http://localhost:8000/#/project?user=1&project=1` on Android browser and get "Error loading project" snackbar. I'm using staging API for the app and webmaker-browser.

**Pull request merged**